packages/office-ui-fabric-react/src/components/Slider/Slider.base.tsx

```diff
@@ -252,8 +298,58 @@ export class SliderBase extends BaseComponent<ISliderProps, ISliderState> implem
     }
     return currentPosition;
   }
+
+  //returns an array of spans each span pertains to a custom label the user passes in
+  private _addLabels(cssRegularLabelClassNames: string | undefined): JSX.Element[] {
+    const { vertical, enableMarks, min, max } = this.props;
+    if (min === undefined || max === undefined) {
+      return [];
+    }
+    // enable marks if an array is an array of JSON with fields of value (denotes where) and label (denotes what to display)
+    const labels: JSX.Element[] = [];
+    if (Array.isArray(enableMarks)) {
```

Comment:
I got rid of this and the code works but the only issue is that the there are red lines under marks in my code because it wants me to make sure that it is an array. I already did check it so I know it has to be a defined array to even reach this method so should I just disregard the red lines?

Reply:
If you disregard the red lines the build will fail